confluent-kafka-mixin/dashboards/confluent-kafka-overview.json

```diff
@@ -298,30 +304,31 @@
         "text": {},
         "textMode": "auto"
       },
-      "pluginVersion": "8.3.3",
+      "pluginVersion": "8.3.5",
       "targets": [
         {
           "datasource": "$datasource",
           "exemplar": true,
-          "expr": "sum(confluent_kafka_server_retained_bytes{scrape_job=~\"$job\",kafka_id=~\"$cluster\",topic=~\"$topic\",partition=~\"$partition\"})/3",
+          "expr": "avg(sum_over_time(confluent_kafka_server_received_records{scrape_job=~\"$job\",kafka_id=~\"$cluster\",topic=~\"$topic\"}[$__rate_interval])/count_over_time(confluent_kafka_server_received_records{scrape_job=~\"$job\",kafka_id=~\"$cluster\",topic=~\"$topic\"}[$__rate_interval])/sum_over_time(confluent_kafka_server_sent_records{scrape_job=~\"$job\",kafka_id=~\"$cluster\",topic=~\"$topic\"}[$__rate_interval])/count_over_time(confluent_kafka_server_sent_records{scrape_job=~\"$job\",kafka_id=~\"$cluster\",topic=~\"$topic\"}[$__rate_interval]))",
```

Comment:
Just double-checking, is this an avg of 3 divisions?

Reply:
Yep it is... I know it looks strange, but I could not think of a better way to calculate the fan-out. If you have any ideas I am open for suggestions :)